src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -10,12 +10,16 @@ use systems::hydraulic::{
     RamAirTurbineController,
 };
 use systems::overhead::{
-    AutoOffFaultPushButton, AutoOnFaultPushButton, FirePushButton, OnOffFaultPushButton,
+    AutoOffFaultPushButton, AutoOnFaultPushButton, FirePushButton, MomentaryPushButton,
+    OnOffFaultPushButton,
 };
 use systems::simulation::{
     SimulationElement, SimulationElementVisitor, SimulatorReader, SimulatorWriter, UpdateContext,
 };
 
+use super::A320EmergencyElectricalOverheadPanel;
+
+use systems::electrical::{consumption::SuppliedPower, ElectricalBusType};
```

Comment:
**suggestion**: Although I'm aware that some of these electrical types currently _must_ be imported. We should try to reduce inter-module dependencies, where e.g. HYD knows about a specific type in ELEC and vice versa. For this reason, I'd suggest adding a trait containing the `rat_and_emer_gen_man_on` fn, and then passing the trait instead of the concrete overhead panel instance.

Overall I try to silo things as much as possible: thus A320 ELEC can use other A320 ELEC types and `systems::electrical` types. A320 HYD can use other A320 HYD types and `systems::hydraulic`. I might move SuppliedPower and ElectricalBusType somewhere else eventually 😅.

Reply:
Did something different from your example I saw after doing it. Just tell me what you think.